app/assets/javascripts/cardboard.js

```diff
@@ -1 +1 @@
-//= require cardboard/admin
\ No newline at end of file
+//= require cardboard/admin
```

Comment:
There's a setting in SublimeText to automatically add newlines to the ends of your files if they're not there. `"ensure_newline_at_eof_on_save": true` over in Default Settings.